hw16_Class_Calculator/Task_1.ts

```diff
@@ -0,0 +1,49 @@
+//Task 1
+// Реализовать класс калькулятор, который принимает несколько методов: сложение, вычитание, умножение, деление
+// и принимает 2 параметра: 2 цифры.
+// Написать 10 положительных и отрицательных тестов по вызову данных функции.
+
+class Calculator {
+
+    constructor(){}
+
+    sum(a: number, b: number){
+        return a + b
+    }
+
+    deduction(a: number, b: number){
+        return a - b
+    }
+
+    multiply(a: number, b: number){
+        return a * b
+    }
+
+    division(a: number, b: number){
+        return a / b
+    }
+
+}
+
+const myCalculator = new Calculator()
+
+const testSum = myCalculator.sum(5,5) === 10 ? 'Test passed' : 'Test failed'
```

Comment:
I suggest to create two functions with console logging, like
const f1 = () => console.log('Test passed');
const f2 = () => console.log('Test failed');
And call these functions in checks.

Reply:
Done.